tnt-persistence/src/main/java/org/dcsa/tnt/persistence/entity/EventCache.java

```diff
@@ -0,0 +1,48 @@
+package org.dcsa.tnt.persistence.entity;
+
+import com.vladmihalcea.hibernate.type.json.JsonBinaryType;
+import lombok.AccessLevel;
+import lombok.AllArgsConstructor;
+import lombok.Builder;
+import lombok.Getter;
+import lombok.NoArgsConstructor;
+import lombok.Setter;
+import org.dcsa.tnt.persistence.entity.enums.EventType;
+import org.hibernate.annotations.Type;
+import org.hibernate.annotations.TypeDef;
+import org.springframework.data.annotation.CreatedDate;
+
+import javax.persistence.Column;
+import javax.persistence.Entity;
+import javax.persistence.EnumType;
+import javax.persistence.Enumerated;
+import javax.persistence.Id;
+import javax.persistence.Table;
+import java.time.OffsetDateTime;
+import java.util.UUID;
+
+@Builder(toBuilder = true)
+@NoArgsConstructor
+@AllArgsConstructor
+@Setter(AccessLevel.PRIVATE)
+@Entity
+@Table(name = "event_cache")
+@Getter
+@TypeDef(name = "jsonb", typeClass = JsonBinaryType.class)
+public class EventCache {
+  @Id
+  @Column(name = "event_id", nullable = false)
+  private UUID eventID;
+
+  @Enumerated(EnumType.STRING)
+  @Column(name = "event_type", nullable = false)
+  private EventType eventType;
+
+  @Type(type = "jsonb")
+  @Column(name = "content", columnDefinition = "jsonb", nullable = false)
+  private String content;
```

Comment:
Can the TO of the event be used for this instead of a String?

Reply:
That would make the pesistence layer dependant on the to layer. So in theory yes, but I'd rather not